app/models/dinkurs_event.rb

```diff
@@ -0,0 +1,87 @@
+class DinkursEvent < ApplicationRecord
```

Comment:
It may be too early to decide this, but if we decide that "local" events have pretty much the same attributes as Dinkur events, then we may want to name this model CompanyEvent, and add a "type" attribute (local or dinkers).

I suspect that local events would look very similar to Dinkurs events.

I would probably change the name to CompanyEvent now, and change/revert to another name later _only_ if there was a reason to do so.

Reply:
I think having them in the same model would be good, as Dinkurs (in my mind) is only used to fetch new/update existing information, as to in the future not having to do so much by hand as a member using DK.